src/components/Jam.jsx

```diff
@@ -46,34 +48,28 @@ export default function Jam() {
   const [collaborativeOperationRunning, setCollaborativeOperationRunning] = useState(false)
   const [schedule, setSchedule] = useState(null)
 
+  // Returns one fresh address for each requested mixdepth.
   const getNewAddresses = useCallback(
```

Comment:
```suggestion
  const getUnusedAddressPerAccount = useCallback(
```

Small nit.

Reply:
Updated it to `getNewAddressesForAccounts`.